static/css/style.css

```diff
@@ -247,6 +247,35 @@ html .all-auth-container {
     text-align: left;
 
 }
+/* Final Score page */
+
+.table_titles + tr {
```

Comment:
These targeted elements might need adjusted as well, if going for a more semantic `<thead>` and `<tbody>` format. (FYI)